index.html

```diff
@@ -3,3 +3,5 @@ <h1>Git Tutorial</h1>
 <p>git diff</p>
 <p>git commit --amend</p>
 <p>git merge</p>
+<p>conflict</p>

```

Comment:
```suggestion
<p>conflict</p>
```suggestion
<p>conflict</p>
Please check blah.
```
```